### DISCUSSION TITLE
修改密码后无法启动

### DISCUSSION THREAD

**Discussion by jintaoo** [apache/doris#18020] [RFC]
使用docker-compose启动了3FE&3BE，启动后可以成功访问doris，通过ALTER USER root@‘%’ IDENTIFIED BY "12345"; 命令修改了root用户密码，密码修改成功，但这时候重启doris就启动不了了，docker-compose down  docker-compose up -d

**Comment by yagagagaga** [comment]
What's your Exception log?

**Comment by crabdave** [comment]
修改脚本中的 -uroot 部分，后面加上新密码 -p12345

/usr/local/bin/entry_point.sh
/usr/local/bin/init_be.sh
